website/docs/r/storage_account.html.markdown

```diff
@@ -49,11 +49,9 @@ The following arguments are supported:
     and `BlobStorage`. Changing this forces a new resource to be created. Defaults
     to `Storage`.
 
-* `account_type` - (Required) Defines the type of storage account to be
-    created. Valid options are `Standard_LRS`, `Standard_ZRS`, `Standard_GRS`,
-    `Standard_RAGRS`, `Premium_LRS`. Changing this is sometimes valid - see the Azure
-    documentation for more information on which types of accounts can be converted
-    into other types.
+* `account_tier` - (Required) Defines the Tier to use for this storage account. Valid options are `Standard` and `Premium`. Changing this forces a new resource to be created
```

Comment:
Wrap the markdown at 80 characters?

Reply:
Taking a look into this - we've got a ton of cases which have gone beyond this limit - so I'm going to leave this as-is for now and we'll do a provider wide cleanup in a bit

Reply:
I know that feeling :wink: :+1: